executor/aggregate_test.go

```diff
@@ -1297,3 +1301,55 @@ func (s *testSuiteAgg) TestIssue20658(c *C) {
 		}
 	}
 }
+
+func (s *testSerialSuite) TestRandomPanicAggConsume(c *C) {
+	tk := testkit.NewTestKitWithInit(c, s.store)
+	tk.MustExec("set @@tidb_max_chunk_size=32")
+	tk.MustExec("set @@tidb_init_chunk_size=1")
+	tk.MustExec("use test")
+	tk.MustExec("drop table if exists t")
+	tk.MustExec("create table t(a int)")
+	for i := 0; i <= 1000; i++ {
+		tk.MustExec(fmt.Sprintf("insert into t values(%v),(%v),(%v)", i, i, i))
+	}
+
+	fpName := "github.com/pingcap/tidb/executor/ConsumeRandomPanic"
+	c.Assert(failpoint.Enable(fpName, `1%panic("ERROR 1105 (HY000): Out Of Memory Quota![conn_id=1]")`), IsNil)
```

Comment:
Why is this a **random** panic?

Reply:
I use `1%` to control the panic frequency.

Reply:
Now I change the panic possible to `5%`, to reduce the retry times.